streams/ar.m3u

```diff
@@ -331,13 +331,45 @@ https://api.new.livestream.com/accounts/679322/events/3782013/live.m3u8
-https://videosenlared.fullstreaming.ar:3501/live/grupocitylive.m3u8
-#EXTINF:-1 tvg-id="TIVTelevision.ar",TIV Televisión (480p) 
-https://inliveserver.com:1936/14044/14044/playlist.m3u8
-#EXTINF:-1 tvg-id="SenalUrbana.ar",Señal Urbana 
+#EXTINF:-1 tvg-id="" status="timeout",La Nación+ (720p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCba3hpU7EFBSk817y9qZkiA/live
+#EXTINF:-1 tvg-id="DoceMisiones.ar" status="online",Doce Misiones
+#EXTVLCOPT:http-user-agent=iPhone
+https://g2.vxral-slo.transport.edge-access.net/a05/ngrp:c12_live01-100129_all/playlist.m3u8
+#EXTINF:-1 tvg-id="CronicaTV.ar" status="online",Cronica TV
+#EXTVLCOPT:http-user-agent=iPhone
+https://g5.vxral-slo.transport.edge-access.net/b16/ngrp:cronicatv_video1-100044_all/playlist.m3u8
+#EXTINF:-1 tvg-id="CMElCanaldelaMusica.ar" status="online",CM El Canal de la Musica
+#EXTVLCOPT:http-user-agent=iPhone
+https://g1.mc-slo.transport.edge-access.net/a09/ngrp:CM_CanaldelaMusica-100044_all/playlist.m3u8
+#EXTINF:-1 tvg-id="CanalC.ar" status="online",Canal C
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.twitch.tv/ccanal
+#EXTINF:-1 tvg-id="Canal4Posadas.ar" status="online",Canal 4 Posadas
+https://iptv.ixfo.com.ar:30443/live/C4POS/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal2.ar" status="online",Canal 2
+https://nd106.republicaservers.com:4433/canal2misioness.mp4
+#EXTINF:-1 tvg-id="Canal10Tucuman.ar" status="online",Canal 10 Tucuman
+https://cdntvstrm.s.llnwi.net/e1/livesec/canal10tuc.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal10MardePlata.ar" status="online",Canal 10 Mar del Plata
+#EXTVLCOPT:http-user-agent=iPhone
+https://g1.mc-hor.transport.edge-access.net/a15/ngrp:canal10mdq-100044_all/playlist.m3u8
+#EXTINF:-1 tvg-id="BarricadaTV.ar" status="online",Barricada TV
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://player.vimeo.com/video/748214742
+#EXTINF:-1 tvg-id="RadioRealpolitik.ar" status="online",Radio Realpolitik
+https://vivo.solumedia.com:19360/realpolitik/realpolitik.m3u8
+#EXTINF:-1 tvg-id="Canal11Machagai.ar" status="online",Canal 11 Machagai
+http://videostream.shockmedia.com.ar:1935/cristalfm/cristalfm/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal29Ensenada.ar" status="online",Canal 29 Ensenada (geo-blocked)
+https://clientes.videostreaming.ar:19360/multimediosensenada/multimediosensenada.m3u8
+#EXTINF:-1 tvg-id="SenalUrbana.ar" status="online",Senal Urbana
 https://videosenlared.fullstreaming.ar:3560/live/senialurbanalive.m3u8
-#EXTINF:-1 tvg-id="NeoxTV.ar",Neox TV 
+#EXTINF:-1 tvg-id="TIVTelevision.ar" status="online",TIV Television
+https://inliveserver.com:1936/14044/14044/playlist.m3u8
+#EXTINF:-1 tvg-id="CiudadMagica.ar" status="online",Ciudad Magica
+https://videosenlared.fullstreaming.ar:3501/live/grupocitylive.m3u8
+#EXTINF:-1 tvg-id="TyCSports.ar" status="online",TyC Sports
+http://line.test-tvsmart.us/play/live.php?mac=00:1A:79:54:57:D2&stream=265093&extension=ts
```

Comment:
Since we do not condone the usage of Xtream-Codes links, i'm afraid this one will not be on the final draft.